docs/manual/java/guide/services/ServiceDescriptors.md

```diff
@@ -80,7 +80,7 @@ A bidirectional streamed call might look like this:
 
 In this case, the server might return a `Source` that converts every message received in the request stream to messages prefixed with `Hello`.
 
-Lagom will choose an appropriate transport for the stream, typically, this will be WebSockets.  WebSockets support bidirectional streaming, and so are a good general purpose option for streaming.  When only one of the request or response message is streamed, Lagom will implement the sending and receiving of the strict message by sending or receiving a single message, and then leaving the WebSocket open until the other direction closes.  Otherwise, Lagom will close the WebSocket when either direction closes.
+Lagom will choose an appropriate transport for the stream, typically, this will be WebSockets.  WebSockets support bidirectional streaming, and so is a good general purpose option for streaming.  When only one of the request or response message is streamed, Lagom will implement the sending and receiving of the strict message by sending or receiving a single message, and then leave the WebSocket open until the other direction closes.  Otherwise, Lagom will close the WebSocket when either direction closes.
```

Comment:
GH UI doesn't highlight the change:

`and so -are- a good general purpose option for streaming.`

`and so +is+ a good general purpose option for streaming.`

Reply:
That you mentioned above was only the change. It seems like there was some grammatical mistake.

Reply:
I wanted to highlight it for future reviewers and myself. :-)

Reply:
Ok sure :-)

Reply:
It still doesn't sound 100% right to me, because the sentence switches from plural to singular:

"WebSockets support bidirectional streaming" <- plural
"and so is a good general purpose option for streaming." <- singular

It probably would be better to rephrase it so that the whole sentence is in agreement, maybe something like: "The WebSocket protocol supports bidirectional streaming, so is a good general purpose option for streaming."

Reply:
yup agree @TimMoore it will sound better.